proposals/csharp-10.0/lambda-improvements.md

```diff
@@ -168,10 +168,10 @@ The [first phase](https://github.com/dotnet/csharplang/blob/main/spec/expression
 > 
 > **An *explicit return type inference* is made *from* an expression `E` *to* a type `T` in the following way:**
 > 
-> *  **If `E` is an anonymous function with explicit return type `Ur` and `T` is a delegate type or expression tree type with return type `Vr` then an *exact inference* ([Exact inferences](expressions.md#exact-inferences)) is made *from* `Ur` *to* `Vr`.**
+> *  **If `E` is an anonymous function with explicit return type `Ur` and `T` is a delegate type or expression tree type with return type `Vr` then an *exact inference* ([Exact inferences](../../spec/expressions.md#exact-inferences)) is made *from* `Ur` *to* `Vr`.**
```

Comment:
This is the link that's currently causing warnings on the docs build. 

We could use the absolute link, but this should work correctly on both.